### ISSUE TITLE
Mixing graphql and withApollo hoc

### ISSUE THREAD

**Issue by cyrilepinat** [apollographql/react-apollo#590]
Hi ! 
Here is my use case:
I have a component that is wrapped in many graphql hoc (one for the list query, another for the delete query, another for the publish query...) trying to make thinks the most reusable as possible. 
In my withDeleteAction hoc I use the updateQueries option to update all lists related to this delete action.
It's working like a charm.
BUT :slightly_smiling_face: 
lately I switched my query list hoc from graphql to withApollo to skip the first load of the query (that I didn't succeed to avoid with the skip config), but my updateQueries are not called anymore...

Is this a bug or it is because graphql and withApollo hoc cannot be mixed? 
thks for your help

### Version
- apollo-client@0.10.1
- react-apollo@0.13.2

**Comment by helfer** [comment]
@cyrilepinat Thanks for filing an issue! Are you sure this is connected to using the `withApollo` HOC? It would seem more likely to me that this was caused by an unrelated change. In either case, it would be very helpful for us to have a reproduction with [react-apollo-error-template](https://github.com/apollographql/react-apollo-error-template) in order to find the (potential) bug and fix it quickly. Thanks!

**Comment by cyrilepinat** [comment]
Hi @helfer, thks for your answer, I would love to find time to create an example but unfortunatelly it's not possible by now. Maybe later. What you say is that using withApollo or graphql hoc from one component to another is possible ? we can mix both ones and they can communicate with eahc other with the same store right ? and updateQueries should be called in any case ? So maybe I did things wrong. Anyway I found a way to resolve my problem by using graphql hoc for all cases so this problem is resolved for me. Thks
